nle/nethack/nethack.py

```diff
@@ -102,25 +100,38 @@ def __init__(
                 "Couldn't find NetHack installation at '%s'." % hackdir
             )
 
-        # Create a HACKDIR for us.
-        self._tempdir = tempfile.TemporaryDirectory(prefix="nle")
-        self._vardir = self._tempdir.name
+        self.shared = False
+
+        if _pynethack.supports_shared():
```

Comment:
Can we also have a flag for enabling/disabling this shared functionality? 

This is for the scenario where we want to have a batch of environments each having its own nhdat file based on the level description file that each env in the batch is periodically updating?